CMakeLists.txt

```diff
@@ -329,6 +330,38 @@ if(CELERITAS_USE_OpenMP)
   find_package(OpenMP REQUIRED)
 endif()
 
+if(CELERITAS_USE_Perfetto)
+  include(FetchContent)
+  if(CELERITAS_USE_CUDA OR CELERITAS_USE_HIP)
+    celeritas_error_incompatible_option(
+      "Perfetto is not supported with CUDA/HIP build"
+      CELERITAS_USE_Perfetto
+      OFF
+    )
+  endif()
+  if (PERFETTO_ROOT AND NOT FETCHCONTENT_SOURCE_DIR_PERFETTO)
+    set(FETCHCONTENT_SOURCE_DIR_PERFETTO ${PERFETTO_ROOT})
+  endif()
+
+  FetchContent_Declare(
+    perfetto
+    # v45 .0
+    URL https://android.googlesource.com/platform/external/perfetto/+archive/7add4fded398ef47414a556b7b81d212a4482a66.tar.gz
+    DOWNLOAD_EXTRACT_TIMESTAMP TRUE
+  )
+
+  if (NOT FETCHCONTENT_SOURCE_DIR_PERFETTO)
+    message(STATUS "Fetching perfetto")
+  endif()
```

Comment:
Does this show up only on the first configure or on every configure? (Also for cmake use `if(` instead of `if (` since it's a function 🙄 )

Reply:
it shows every time, I removed it